source/api/commands/type.md

```diff
@@ -259,6 +258,26 @@ cy.get('input[type=text]').type('Test all the things', { force: true })
 
 # Notes
 
+## Supported Elements
+
+* HTML `<body>` and `<textarea>` elements.
+* Elements with a defined `tabindex` attribute.
+* Elements with a defined `contenteditable` attribute.
+* HTML `<input>` elements with a defined `type` attribute of one of the following:
+  * `text`
+  * `password`
+  * `email`
+  * `number`
+  * `date`
+  * `week`
+  * `month`
+  * `time`
+  * `datetime`
+  * `datetime-local`
+  * `search`
+  * `url`
+  * `tel`
+
```

Comment:
It proper looks like I've just copied your -- Jennifer's -- copy from the ticket here... and I have 😊

I thought about adding some surrounding text but on reflection that seemed superfluous. This is a list of the supported elements: no fluff, just stuff which is what I want in reference material.